mmv1/products/databasemigrationservice/terraform.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022 Google Inc.
```

Comment:
`terraform.yaml` will still work after https://github.com/GoogleCloudPlatform/magic-modules/pull/7463, but not preferred going forward, and will be stripped eventually.

Lots of overrides here. If you want to try [my converting script](https://github.com/GoogleCloudPlatform/magic-modules/compare/main...c2thorn:magic-modules:yaml-script) you are welcome to set it up (change the `folders` variable to the relative path of your `products/databasemigrationservice` folder, install `ruamel.yaml` in your python env)

Not sure if that's faster than copy/pasting the values manually for this size.